lib/matplotlib/projections/polar.py

```diff
@@ -1161,34 +1159,84 @@ def set_theta_direction(self, direction):
     def get_theta_direction(self):
         """
         Get the direction in which theta increases.
-
+        
         -1:
            Theta increases in the clockwise direction
-
+        
         1:
            Theta increases in the counterclockwise direction
         """
         return self._direction.get_matrix()[0, 0]
 
     def set_rmax(self, rmax):
+        """Update the maximum radius
+        
+        Parameters
+        ----------
+        rmax : number
+            Set maximum radius to value rmax.
+        """
         self.viewLim.y1 = rmax
 
     def get_rmax(self):
+        """
+        Returns
+        -------
+        float
+            The maximum radius of chart.
+        """
         return self.viewLim.ymax
 
     def set_rmin(self, rmin):
+        """Updates the minimum radius
+        Parameters
+        ----------
+        rmin : number
+            Set minimumradius to value rmin.
+        """
         self.viewLim.y0 = rmin
 
     def get_rmin(self):
+        """
+        Returns
+        -------
+        float
+            The minimum radius of chart.
+        """
         return self.viewLim.ymin
 
     def set_rorigin(self, rorigin):
+        """Updates the chart origin
+        
+        Parameters
+        ----------
+        rorigin : number
+            Set the chart origin value.
+        """
         self._originViewLim.locked_y0 = rorigin
 
     def get_rorigin(self):
+        """
+        Returns
+        -------
+        float
+            The origin of chart.
+        """
         return self._originViewLim.y0
 
     def set_rlim(self, *args, **kwargs):
+
+         """Set the minimum and maximum radius.
+         
+        Parameters
+        ----------
+        rmin = 0
```

Comment:
This should be `rmin : float, optional`, and then the default value should be mentioned in the sentence below. I'm not sure there is a default value though, so here I think the 0 can just be omitted.

Reply:
Thank you for your patience as this my first PR!  Hopefully I will make future PR's much faster.  Will finish this today.